lib/logstash/filters/prune.rb

```diff
@@ -91,13 +91,15 @@ def filter(event)
     unless @whitelist_names.empty?
       @whitelist_names_regexp = Regexp.union(@whitelist_names.map {|x| Regexp.new(event.sprintf(x))}) if @interpolate
       hash.each_key do |field|
+        next if field.nil?
```

Comment:
can we add a log line for this behaviour? specially when we aim to debug this behaviour why is not creating the fields the user is expecting to?